ui/js/controllers/logviewer.js

```diff
@@ -33,13 +32,58 @@ logViewerApp.controller('LogviewerCtrl', [
         $scope.currentLineNumber = 0;
         $scope.highestLine = 0;
         $scope.showSuccessful = true;
-
+        $scope.willScroll = false;
         $scope.$watch('artifact', function () {
```

Comment:
Can we add an empty line, just above this `$scope.$watch` line if that's ok? It helps readability a little.